src/toICN-after.js

```diff
@@ -45,6 +54,37 @@ function waitElement(webSiteName, cb) {
   }, 300);
 }
 
+let barText = '';
+barText += '<div class="toicnbar" style="background-color: #f4ffa2; margin-top: 5px; margin-bottom: 5px; padding: .75rem 1.25rem;">';
```

Comment:
marginもpaddingと同様に一気に指定するのはどうでしょう？

```suggestion
barText += '<div class="toicnbar" style="background-color: #f4ffa2; margin: 5px auto; padding: .75rem 1.25rem;">';
```

Reply:
修正しました。